### DISCUSSION TITLE
Haven't been approved yet

### DISCUSSION THREAD

**Discussion by emilytin0206** [community/community#130966] [GitHub Education]
### Hi everyone,

Hi! I applied for use copilot on 7/1 , now looks like I haven't been approved yet :)

**Comment by davevad93** [comment]
Hi @emilytin0206 ,

It usually takes only a few days for benefits to become available, but sometimes it can take up to a couple of weeks. The waiting time can vary depending on a number of factors, such as the volume of applications.

When the benefits are available the status will change to:

`Congrats! Your request for @emilytin0206 was approved and your benefits are now available`
